app/Domain/Auth/Templates/login.blade.php

```diff
@@ -56,7 +58,7 @@
         <?php $tpl->dispatchTplEvent('beforeSubmitButton'); ?>
 
         <div class="">
-            <x-global::forms.button type="submit" content-role="primary" name="login">
+            <x-global::forms.button type="submit" content-role="primary" name="login" cornerShape="full" :takeFullWidth="true">
```

Comment:
I assume the cornershape about the rounded corners? Let's take this out and keep the buttons consistent.

Reply:
class="w-full"

Reply:
done